src/cerebral.d.ts

```diff
@@ -0,0 +1,107 @@
+﻿declare module "cerebral" {
```

Comment:
we shouldn't ship old style global modules definitions. I'll exclude this commit for now. Anyway thanks for initial typings. I'll prepare new setup based on this.